src/header/css/header-menus.pcss

```diff
@@ -396,6 +396,11 @@
 							right: 0;
 							top: 0;
 							width: 1px;
+
+							@nest :dir(rtl) & {
+								left: 25px;
```

Comment:
was there a reason you're not using the logical properties plugin to do the rtl here?

Reply:
@paulcpederson logical properties plugin?

Reply:
yeah postcss is running this plugin: https://github.com/csstools/postcss-logical

Basically it's a different property name that automatically transitions from right to left. So instead of `left` you use `inset-inline-start` and it does the right to left flip for you.

Here is more details: https://css-tricks.com/css-logical-properties/